components/admin_console/group_settings/group_details/group_details.jsx

```diff
@@ -51,17 +52,20 @@ export default class GroupDetails extends React.PureComponent {
             loadingTeamsAndChannels: true,
             addTeamOpen: false,
             addChannelOpen: false,
+            allowReference: Boolean(props.group.allow_reference),
         };
     }
 
     componentDidMount() {
-        const {groupID, actions} = this.props;
+        const {groupID, actions, group} = this.props;
         actions.getGroup(groupID);
+
         Promise.all([
             actions.getGroupSyncables(groupID, Groups.SYNCABLE_TYPE_TEAM),
             actions.getGroupSyncables(groupID, Groups.SYNCABLE_TYPE_CHANNEL),
         ]).then(() => {
             this.setState({loadingTeamsAndChannels: false});
+            this.setState({group, allowReference: Boolean(this.props.group.allow_reference)});
```

Comment:
"Duplicate" call to `setState` here -- I suspect this is a non-issue, but just highlighting in case it makes more sense to merge with the line above.